components/esp_rainmaker/include/esp_rmaker_ota.h

```diff
@@ -83,6 +103,8 @@ typedef struct {
     char *priv;
     /** OTA Metadata. Applicable only for OTA using Topics. Will be received (if applicable) from the backend, along with the OTA URL */
     char *metadata;
+    /** The Function to be called for reporting OTA status. This can be used if needed to override transport of OTA*/
+    esp_rmaker_ota_report_fn_t report_fn;
```

Comment:
This is a breaking change if I'm not mistaken. Please, increase the component version and update in the dependencies accordingly.
Please add a line in the CHANGES.md as well